core/src/main/resources/org/kohsuke/stapler/bind.js

```diff
@@ -26,39 +26,43 @@ function makeStaplerProxy(url,crumb,methods) {
             for (var i=0; i<args.length-(callback!=null?1:0); i++)
                 a.push(args[i]);
 
-            if(window.jQuery === window.$) { //Is jQuery the active framework?
```

Comment:
Let it be said that this dead jQuery code path would have never worked with Jenkins. `progressiveRendering.jelly` expects a JSON response and invokes `callback.responseObject()` to get the JSON. `renderOnDemand.jelly` expects HTML and invokes `callback.responseText` to get the HTML. This dead jQuery code path was only defining `responseObject`, so it would have worked with `progressiveRendering.jelly` but not `renderOnDemand.jelly`.